python/cugraph/structure/symmetrize.py

```diff
@@ -16,7 +16,7 @@
 import dask_cudf
 
 
-def symmetrize_df(df, src_name, dst_name):
+def symmetrize_df(df, src_name, dst_name, multi=False, symmetrize=True):
```

Comment:
Can you update the docstring with the new args?  I'm especially interested in the expected usage of the `symmetrize` arg, since at first glance it seems odd to call symmetrize_df with a flag telling it not to symmetrize.

Reply:
Added doc. This might be updated in a symmetrize PR to accept graphs